### ISSUE TITLE
DroneDeploy Segmentation Dataset

### ISSUE THREAD

**Issue by andreaschandra** [jakartaresearch/earth-vision#22]
Add semantic segmentation dataset
https://github.com/chrieke/awesome-satellite-imagery-datasets#3-semantic-segmentation

- [DroneDeploy Segmentation Dataset](https://github.com/dronedeploy/dd-ml-segmentation-benchmark) (DroneDeploy, Dec 2019)
Drone imagery (0.1m res., RGB), labels (7 land cover catageories: building, clutter, vegetation, water, ground, car) & elevation data, baseline model implementation.

**Comment by rubentea16** [comment]
https://arxiv.org/pdf/1704.06857.pdf

**Comment by rubentea16** [comment]
TO DO:

- [ ] Download dataset code
- [ ] Convert raw dataset to tensor PyTorch code  (ref: https://github.com/dronedeploy/dd-ml-segmentation-benchmark/blob/master/libs/datasets_keras.py)

**Comment by rubentea16** [comment]
Notes :  Ada 2 dataset type yaitu  'dataset-sample'  (0.5 GB) dan 'dataset-medium'  (9.0 GB)

Untuk dataset-sample hanya 3 file yang digunakan, setiap file ini akan di convert menjadi  `chips` 
- 1d4fbe33f3_F1BE1D4184INSPIRE-ortho.tiff => Train
- ec09336a6f_06BA0AF311OPENPIPELINE-ortho.tiff => Validation
- e1d3e6f6ba_B4DE0FB544INSPIRE-ortho.tiff => Test

**Comment by rubentea16** [comment]
https://github.com/dronedeploy/dd-ml-segmentation-benchmark/blob/284f729899495b2ea853d3c364155dd0d5cae56e/libs/inference_keras.py#L23

function `chips_from_image` =>  convert image ke chips tapi store-nya di variable List (tidak write chips image-nya ke dalam folder)

**Comment by rubentea16** [comment]
@andreaschandra dre, scope issue ini sampe jadi data loader doang?

**Comment by andreaschandra** [comment]
> @andreaschandra dre, scope issue ini sampe jadi data loader doang?

iya, scope ini sampe dataloader aja, dan kalo datasetnya provide banyak task, bisa di specify pake class name atau method buat generate data untuk task yang mana e.g chip classification, object detection, instance segmentation

**Comment by andreaschandra** [comment]
@gilangarisptr

**Comment by rubentea16** [comment]
@gilangarisptr baca issue ini yaa
